lighthouse-core/test/computed/computed-artifact-test.js

```diff
@@ -12,7 +12,7 @@ const assert = require('assert');
 const makeComputedArtifact = require('../../computed/computed-artifact.js');
 
 describe('ComputedArtifact base class', () => {
-  it.skip('caches computed artifacts by strict equality', async () => {
+  it('caches computed artifacts by strict equality', async () => {
     let computeCounter = 0;
 
     const TestComputedArtifact = makeComputedArtifact(class TestComputedArtifact {
```

Comment:
class as expression hurts my head, especially since it has the same name as something different
can you extract?